libgpos/src/memory/CMemoryPoolManager.cpp

```diff
@@ -93,12 +94,45 @@ CMemoryPoolManager::CMemoryPoolManager
 //
 //---------------------------------------------------------------------------
 GPOS_RESULT
-CMemoryPoolManager::EresInit()
+CMemoryPoolManager::EresInit
```

Comment:
We had an offline discussion about this which resulted to removing CMemoryPoolMalloc all together and use a CMemoryPoolAlloc that can be initialized with malloc/free in gpos_init()